bin/cli.js

```diff
@@ -48,7 +47,6 @@ const cliFlags = yargs
   })
   .option('config', {
     'describe': 'Path to json config file',
-    'coerce': (arg) => getConfigFromFile(arg),
```

Comment:
I'm 👍  for keeping value of config as `string`. It's more consistent and architecturaly right. Below we read config from file and mix it with cli options. No magic, very simple to read and operate with config values.